tst/com/amazon/ion/benchmark/DataGeneratorTest.java

```diff
@@ -126,6 +133,43 @@ public static void violationDetect(String inputFile) throws Exception {
         }
     }
 
+    /**
+     * This method executes Ion Data Generation twice and compare the generated data then provide the comparison result which represents by the boolean value.
+     * @param seedOption represents whether the '--seed' option is specified. If it is specified, the value of this option is 'true'. Otherwise, the value is 'false'.
+     * @return Boolean value which represents the comparison result. If there is any difference between two generated ion files, the return value will be set to 'false'.
+     * @throws Exception if there is error when executing data generation.
+     */
+    public Boolean generateAndCompare(Boolean seedOption) throws Exception {
+        Boolean result = true;
+        for (File testFile : TEST_ISL_FILES) {
+            String testFilePath = testFile.getPath();
+            for (int i = 0; i < 2; i++) {
+                String outputFile = "testSeed" + i + ".ion";
+                Map <String, Object> optionsMap = Main.parseArguments("generate", "--data-size", "5000", "--format", "ion_text", "--input-ion-schema", testFilePath, outputFile);
+                if (seedOption == true) {
+                    optionsMap = Main.parseArguments("generate", "--data-size", "5000", "--seed", "200", "--format", "ion_text", "--input-ion-schema", testFilePath, outputFile);
+                }
```

Comment:
Consider using if/else to avoid redundantly calling `parseArguments` when `seedOption == true`.

Reply:
> Consider using if/else to avoid redundantly calling `parseArguments` when `seedOption == true`.

Yes, that makes more sense. Will update it in the following commit.